WordPress/src/main/res/layout/edit_post_settings_fragment.xml

```diff
@@ -220,6 +220,30 @@
 
         </LinearLayout>
 
+        <!-- Mark as Sticky -->
+
+        <LinearLayout
+            android:id="@+id/post_settings_mark_as_sticky_container"
+            style="@style/PostSettingsCardViewInnerLayout" >
+
+            <com.google.android.material.textview.MaterialTextView
+                style="@style/PostSettingsSectionHeader"
+                android:text="@string/post_settings_mark_as_sticky_options_header" />
+
+            <LinearLayout style="@style/PostSettingsContainer">
+
+                <androidx.appcompat.widget.SwitchCompat
+                    android:id="@+id/post_settings_sticky_switch"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="@string/post_settings_post_sticky"
+                    android:textAlignment="viewStart"
+                    android:textAppearance="?attr/textAppearanceSubtitle1" />
```

Comment:
What do you think about moving these attributes to `styles.xml` similarly to some of the other components here?

Reply:
Make sense. My thought before was that it was used in a single place, so I decided to not extract them.

I just pushed a commit with them extracted to `styles.xml` 😄